doc/source/conf.py

```diff
@@ -409,9 +408,9 @@ def add_custom_assets(
     if pagename == "train/train":
         app.add_css_file("css/ray-train.css")
     elif pagename == "index":
-        # CSS for HTML part of index.html
-        app.add_css_file("css/splash.css")
-        app.add_js_file("js/splash.js")
+        app.add_css_file("css/index.css")
+        app.add_js_file("js/index.js")
+        return "index.html"
```

Comment:
why is this returning `"index.html"` here?

Reply:
If you return a string from a function triggered by an `html-page-context` event, sphinx will use that template instead of the default `page.html` template for that page. [There's a note about this in the docstring for that function](https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx-core-events), but I can add more explanation if that would be helpful.

Reply:
thanks for the explanation. could you move the index page handling up into its own section, and add a comment:

````
if pagename == "index":
  app.add_css_file("css/index.css")
  app.add_js_file("js/index.js")
  return "index.html"  # uses a special template

# The following uses the default page.html template
if pagename == "train/tarin":
  ...
````